sequencing/vcf_qc/Readme.md

```diff
@@ -1,21 +1,19 @@
-<!-- dx-header -->
 # VCF QC (DNAnexus Platform App)
 
-Calls [GATK ApplyRecalibration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.php) and [GATK VariantFiltration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.php) to apply filters to VCF files.
+## What does this applet do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app uses [GATK ApplyRecalibration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.php) and [GATK VariantFiltration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.php) to apply filters to VCF files.
 
+## What data are required for this app to run?
 
+This app requires:
+1. The array of VCF (variant calling format) files (`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`). VCF is the standard format for variant calling.
```

Comment:
Do we have to define VCF here? Wouldn't the target audience of this app know this already?